.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,9 +10,16 @@
 - Example of usage (if applicable)
 - Example of changes to config files (if applicable)
 *
+-->
+
 ### Checklist
 - [ ] The code change is tested and works locally.
-- [ ] Tests pass. Your PR cannot be merged unless tests pass
+- [ ] Tests pass. Your PR cannot be merged unless tests pass. -- `behave`
```

Comment:
I think adding the commands here is a good idea, but we should use the poetry or make commands here, since that's what's recommended in our documentation.